core/templates/dev/head/css/oppia.css

```diff
@@ -1055,7 +1055,7 @@ textarea {
   background: linear-gradient(to right, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
   /*The display is set in Library.js*/
   display: none;
-  height: 240px;
+  height: 259px;
```

Comment:
Re 258/259: Gradient/shadow can't really be adjusted, but since it is a _very_ minor rendering issue, I'm happy to kick it back to 258. 

Re duplication:
I've also tried moving the background up to the combined class and ordering of the CSS (separaretely) which yielded the same results. 
```css
.oppia-library-carousel-overlay-left,
.oppia-library-carousel-overlay-right {
  /*The display is set in Library.js*/
  display: none;
  height: 259px;
  left: 0;
  margin-top: 12px;
  position: absolute;
  top: 0;
  width: 20px;
  z-index: 99;
}

.oppia-library-carousel-overlay-left {
  background: #b5bdc8;
  background: -moz-linear-gradient(left, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
  background: -webkit-linear-gradient(left, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
  background: linear-gradient(to right, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
}

.oppia-library-carousel-overlay-right {
  background: #b5bdc8;
  background: -moz-linear-gradient(left, rgba(0,0,0,0) 0%, rgba(0,0,0,0.65) 100%);
  background: -webkit-linear-gradient(left, rgba(0,0,0,0) 0%, rgba(0,0,0,0.65) 100%);
  background: linear-gradient(to right, rgba(0,0,0,0) 0%, rgba(0,0,0,0.65) 100%);
}
```